homeassistant/components/squeezebox/media_player.py

```diff
@@ -325,7 +325,7 @@ async def async_will_remove_from_hass(self) -> None:
     @property
     def volume_level(self) -> float | None:
         """Volume level of the media player (0..1)."""
-        if self._player.volume:
+        if self._player.volume or self._player.volume == 0:
```

Comment:
```suggestion
        if self._player.volume is not None:
```
I think this is what you are looking for?